src/main/webapp/resources/js/pages/announcement/EditAnnouncement.jsx

```diff
@@ -0,0 +1,35 @@
+import React, { useRef } from "react";
```

Comment:
Comments

Reply:
Updated in 6a78da0940b3065c745f3e4d7579bc85e85187c8